index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style/style.css" />
+  </head>
+  <body id="main">
+    <title id="title">Dr. Norman Borlaug</title>
+    <div id="img-div">
+      <h1>Dr. Norman Borlaugs</h1>
+      <img class="responsive" src="/img/Dr Norman.jpg" alt="Norman Borlaug" />
+      <caption id="img-caption">
```

Comment:
The image here does not display on your webpage, check to be sure that your img tag links to the right folder.